### PR TITLE
changed all class based components to functional based

### PR DESCRIPTION
**Pull request by A00020061**
All class based components have been changed into functional based components.

### CODE REVIEW

File: src/Components/Charts/charts.js [hunk 1]
```diff
@@ -13,48 +13,52 @@ import {
...
+    <Container>
+      <MainContainer>
+        <ChartContainer>
+          <CoinInfoDiv>
+            <DisplayChartNavbar
+              {...props}
+              // func={this.props.func}
+              // coinList={this.props.coinList}
+              // alert={this.props.alert}
+              // alert2={this.props.alert2}
```
Comment (yoloOnTheBattlefield) [inline src/Components/Charts/charts.js:36]: remove this

File: src/Components/Navbar/index.styles.js [hunk 2]
```diff
@@ -27,17 +28,17 @@ export const SearchDiv = styled.div`
   width: 75%;
   box-shadow: 0px 0px 0.5px 0.5px #fafbfb;
   justify-content: space-between;
-  @media only screen and (min-width: 1400px) {
+  /* @media only screen and (min-width: 1400px) {
     width: 82%;
-  }
+  } */
```
Comment (yoloOnTheBattlefield) [inline src/Components/Navbar/index.styles.js:33]: remove this

File: src/Components/Navbar/index.styles.js [hunk 3]
```diff
@@ -27,17 +28,17 @@ export const SearchDiv = styled.div`
...
 export const Wrapper = styled.div`
   display: flex;
-  width: 22%;
+  width: 20%;
   justify-content: space-between;
-  @media only screen and (min-width: 1400px) {
+  /* @media only screen and (min-width: 1400px) {
     width: 13%;
-  }
+  } */
```
Comment (yoloOnTheBattlefield) [inline src/Components/Navbar/index.styles.js:41]: and this

File: src/Pages/CoinPage/CoinPage.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  const [isLoading, setIsLoading] = useState(false);
+  useEffect(() => {
+    getCoinData();
+  });
+  const getCoinData = async () => {
+    try {
+      const response = await axios(
+        `https://api.coingecko.com/api/v3/coins/bitcoin?localization=false&tickers=false&market_data=true&community_data=true&developer_data=false&sparkline=false`
+      );
+      console.log("hello", response.data);
```
Comment (yoloOnTheBattlefield) [inline src/Pages/CoinPage/CoinPage.js:17]: remove this

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/Components/Charts/charts.js (hunk 1 above)
Comment (A00020061) [reply src/Components/Charts/charts.js:36]: removed

File: src/Components/Navbar/index.styles.js (hunk 2 above)
Comment (A00020061) [reply src/Components/Navbar/index.styles.js:33]: removed

File: src/Components/Navbar/index.styles.js (hunk 3 above)
Comment (A00020061) [reply src/Components/Navbar/index.styles.js:41]: removed

File: src/Pages/CoinPage/CoinPage.js (hunk 4 above)
Comment (A00020061) [reply src/Pages/CoinPage/CoinPage.js:17]: removed

**Pull request merged**